### PR TITLE
[wrapper] improve(style): theme wiring

### PR DESCRIPTION
**Pull request by Barackos**
Click below to open examples:
rich-content: https://rich-content-gallery-settings-theme-finals.surge.sh/
rich-content-storybook: https://rich-content-storybook-gallery-settings-theme-finals.surge.sh/

### CODE REVIEW

File: packages/editor-common/web/statics/styles/radio-group.scss [hunk 1]
```diff
@@ -2,14 +2,18 @@
   cursor: pointer;
   font-size: 14px;
   line-height: 1.5;
+
+  &:hover .radioGroup_button {
```
Comment (lxgreen) [inline packages/editor-common/web/statics/styles/radio-group.scss]: do you mean that every radio button border should be black when a group is hovered?

File: packages/editor-common/web/statics/styles/color-picker.scss [hunk 2]
```diff
@@ -33,7 +33,6 @@
   cursor: pointer;
   outline: 0;
   padding: 0;
-  margin-left: 12px;
```
[the remark this answers is not in the document]
Comment (lxgreen) [reply packages/editor-common/web/statics/styles/color-picker.scss]: image

File: packages/editor-common/web/statics/styles/selection-list.scss [hunk 3]
```diff
@@ -11,10 +11,23 @@
 .selectionListOption {
   font-size: 14px;
   text-align: center;
-  margin: 0 auto 0 auto;
   cursor: pointer;
+  width: 100%;
```
Comment (lxgreen) [inline packages/editor-common/web/statics/styles/selection-list.scss]: this change: 
image

Comment (lxgreen) [reply packages/editor-common/web/statics/styles/selection-list.scss]: master:
image

File: packages/plugin-button/web/src/components/button-component.jsx [hunk 4]
```diff
@@ -21,7 +21,6 @@ class ButtonComponent extends PureComponent {
     const rel = button.settings.rel ? 'nofollow' : '';
     const style = {
       border: '0px solid blue',
-      ...this.props.style,
```
Comment (lxgreen) [inline packages/plugin-button/web/src/components/button-component.jsx]: please elaborate on this

File: packages/plugin-gallery/web/statics/styles/gallery-items-sortable.scss [hunk 5]
```diff
@@ -109,6 +109,9 @@
   &:active {
     cursor: -webkit-grabbing;
```
Comment (lxgreen) [inline packages/plugin-gallery/web/statics/styles/gallery-items-sortable.scss]: supported on chrome only

File: packages/editor-common/web/statics/styles/selection-list.scss (hunk 3 above)
Comment (Barackos) [reply packages/editor-common/web/statics/styles/selection-list.scss]: fixed on [thumbnail-placement-selector.rtlignore.scss](https://github.com/wix-incubator/rich-content/pull/828/files#diff-8bd72f80d4953150754dfa0db66a07c1), Line 12:

`grid-template-columns: auto auto auto auto;`

File: packages/editor-common/web/statics/styles/radio-group.scss (hunk 1 above)
Comment (Barackos) [reply packages/editor-common/web/statics/styles/radio-group.scss]: indeed

File: packages/plugin-text-color/web/src/toolbar/TextColorPanel.jsx [hunk 6]
```diff
@@ -51,7 +51,7 @@ export default class TextColorPanel extends Component {
       editorState = this.getInlineColorState(colorName);
     this.setState({
       currentColor: newColorHex || defaultColor,
-      currentSchemeColor: this.currentColors[0] && getColor(this.currentColors[0]),
+      currentSchemeColor: newColorHex || (this.currentColors[0] && getColor(this.currentColors[0])),
```
[the remark this answers is not in the document]
Comment (Barackos) [reply packages/plugin-text-color/web/src/toolbar/TextColorPanel.jsx]: It's a case that should be treated.
Should have considered it in a [recent fix I made](https://github.com/wix-incubator/rich-content/commit/6f381857ec4144f75c11d78465bf5b069fc6e530#diff-6980d99be9ec53a868c9360e4a7e945f) in this code

File: packages/plugin-button/web/src/constants.js [hunk 7]
```diff
@@ -46,9 +47,40 @@ export const getDefaultComponentData = (rel, target) => {
...
-        borderColor: COLORS.color8,
+        background: getColors().color8,
+        color: getColors().color1,
+        borderColor: getColors().color8,
+      },
+    },
+  };
+};
+
+const WRAPPER_PALETTE = {};
```
Comment (lxgreen) [inline packages/plugin-button/web/src/constants.js]: what is bothering me in this file, is its initial inconsistency (before your changes). the `constants` module supposed to contain constants only. the utils like `getDefaultComponentData`, `getColors` should be placed in `utils`. the `const THEME` is super confusing -- it is not constant and not a static object -- it is a function that accepts parameters. should move to utils as well.

**Pull request merged**